pkg/controllers/csi/provisioner/controller_test.go

```diff
@@ -42,7 +42,7 @@ func (fs *mkDirAllErrorFs) MkdirAll(_ string, _ os.FileMode) error {
 	return fmt.Errorf(errorMsg)
 }
 
-func TestOneAgentProvisioner_Reconcile(t *testing.T) { //nolint:revive
+func TestOneAgentProvisioner_Reconcile(t *testing.T) {
 	ctx := context.TODO()
```

Comment:
think in unit-tests context.Background() is used

Reply:
Fixed in https://github.com/Dynatrace/dynatrace-operator/pull/2710/commits/560727953ee8b7e6a577db07967421ac7ee56b5f